docs/book/src/SUMMARY.md

```diff
@@ -102,6 +102,7 @@
           - [Cluster Infrastructure](./developer/providers/cluster-infrastructure.md)
           - [Machine Infrastructure](./developer/providers/machine-infrastructure.md)
           - [Bootstrap](./developer/providers/bootstrap.md)
+          - [Control Plane](./developer/providers/control-plane.md)
```

Comment:
nit: I would move this up, below Cluster Infrastructure (L103)

Reply:
👍  , Addressed in [39919b5](https://github.com/kubernetes-sigs/cluster-api/pull/11113/commits/39919b5eb2ade2ab5bef28c55f1a0e16d9854f38). Thanks